src/core_plugins/console/api_server/es_6_0/query.js

```diff
@@ -25,47 +25,47 @@ var SPAN_QUERIES = {
   }
 };
 
-var DECAY_FUNC_DESC = {
+const DECAY_FUNC_DESC = {
+  __template: {
+    'FIELD': {
+      'origin': '',
+      'scale': ''
+    }
+  },
+  '{field}': {
+    'origin': '',
+    'scale': '',
+    'offset': '',
+    'decay': 0.5
+  }
+};
+const SCORING_FUNCS = {
+  'script_score': {
     __template: {
-      "FIELD": {
-        "origin": "",
-        "scale": ""
-      }
+      'script': '_score * doc[\'f\'].value'
     },
-    "{field}": {
-      "origin": "",
-      "scale": "",
-      "offset": "",
-      "decay": 0.5
+    'script': {
+      //populated by a global rule
     }
   },
-  SCORING_FUNCS = {
-    "script_score": {
-      __template: {
-        "script": "_score * doc['f'].value"
-      },
-      "script": {
-        //populated by a global rule
-      }
-    },
-    "boost_factor": 2.0,
-    "random_score": {
-      "seed": 314159265359
+  'boost_factor': 2.0,
```

Comment:
Perhaps we should add the `quote-props: ['error', 'as-needed']` to our eslint config, but these keys don't need them.

Thanks for switching things to single quotes btw.